### PR TITLE
Converted to use Selenium 2

### PR DESCRIPTION
**Pull request by wright-io**
It's done!!!  Phil: Please check it out and let me know if you see any reason not to merge with master.

### CODE REVIEW

File: content/landing_page.md [hunk 1]
```diff
@@ -161,12 +164,11 @@ See Sauce Labs for the list of available browser/OS configurations.
...
 
 ## Design Sources
-In addition to being built on top op [Sauce Labs](http://saucelabs.com/), 
-Selenium, and [Soda](https://github.com/LearnBoost/soda), `Given When Then` is heavily 
-influenced by [Cucumber](http://cukes.info/) and the 
-[BDD movement](http://en.wikipedia.org/wiki/Behavior_Driven_Development) 
-in general - and in turn all of the efforts that BDD and Cucumber 
-are indebted to.  It's cucumbers all the way down baby!
+In addition to being built on top op [Sauce Labs](http://saucelabs.com/) 
+and Selenium 2, `Given When Then` is heavily influenced by [Cucumber](http://cukes.info/)
```
Comment (philcockfield) [inline content/landing_page.md:168]: typo - op => "of"

Comment (wright-io) [inline content/landing_page.md:168]: Fixed.

File: lib/driver.coffee [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+
+
+    ###
+    Convenience method to assert text is present on the current page.
+    @param text:      Text to search for.
+    @param callback:  Callback.
+    @option inverse:  Whether to inverse the result (assert text is not present).
+    ###
+    assertTextPresent: (text, opts=[], callback) ->
+      # TODO: Again, need a way to not need to pass opts.
```
Comment (philcockfield) [inline lib/driver.coffee:169]: Shouldn't `opts` be?:

`opts = {}`

**Comment by philcockfield** [comment]
Merge away.

File: lib/driver.coffee (hunk 2 above)
Comment (wright-io) [inline lib/driver.coffee:169]: Oops - that was dumb.  Fixed.

**Pull request merged**